es6-destructuring/index.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>ES6 Destructuring</title>
+  </head>
+  <body>
+    <script>
+      const book = {
+        title: 'Goodnight Punpun',
+        author: 'Inio Asano',
+        libraryID: 3353,
+      };
+
+      const { title: t, author: a, libraryID: id } = book;
```

Comment:
You do not need the variables `t`, `a`, or `id`. You only need to create a different variable name or `alias` if you already have variables with those names. Otherwise you can just use the property name